tee-worker/litentry/primitives/src/aes.rs

```diff
@@ -20,7 +20,8 @@ extern crate sgx_tstd as std;
 #[cfg(all(not(feature = "std"), feature = "sgx"))]
 extern crate rand_sgx as rand;
 
-use crate::{AesOutput, UserShieldingKeyType, USER_SHIELDING_KEY_NONCE_LEN};
+use crate::{AesOutput, UserShieldingKeyType};
+use ring::aead::NONCE_LEN;
```

Comment:
You probably want to expose it in `primitives/core/src/key.rs` and import it in this crate and `use` it from the crate, similar to e.g. `AesOutput`